lib/gollum/public/gollum/javascript/mermaid.js

```diff
@@ -0,0 +1,2 @@
+// Load mermaid from installed Node module
```

Comment:
Created this separate JS file instead of adding to `app.js` so we can require the mermaid library conditionally (so we don't load it when mermaid isn't active anyway)